app/src/main/java/com/cvtracker/vmd/home/MainPresenter.kt

```diff
@@ -156,7 +155,6 @@ class MainPresenter(override val view: MainContract.View) : AbstractCenterPresen
             PrefHelper.favEntry = searchEntry
             view.showCenters(emptyList(), null)
         }
-        selectedSortType = null
         resetFilters(needRefresh = false)
```

Comment:
If we decide to save filters, we should remove this useless line

Reply:
Humm, it's a delete change, so it's done. 🤔

Reply:
No I talk about `resetFilters(needRefresh = false)` :)

Reply:
My mistake 😞, i take the point